lib/krane/api_resource.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Krane
+  class APIResource
```

Comment:
Its only for containing the response of the api resources call, so I don't have to return a hash.

Reply:
Ah, in that case we shouldn't need it, because per my other comment we should be using the discovery information directly instead of re-making the calls via kubectl and parsing tabular output. But if that really isn't possible for some reason, I'd make this class private to the class that needs it, to avoid confusion.